index.js

```diff
@@ -0,0 +1,122 @@
+    artistData('Carol G', ['Bichota', 'Tusa (with Nicki Minaj)', 'Ay, DiOs Mío!'], 'A rising reggaeton star celebrated for her dynamic vocals and empowering presence.', 'https://remezcla.com/wp-content/uploads/2023/07/header-GettyImages-1529560337-1424x1068.jpg'),
+    artistData('Bad Bunny', ['Dákiti (with Jhay Cortez)', 'Vete', 'Mía (with Drake)'], "A genre-defying phenomenon who's reshaped Latin music with his genre-blending creativity and chart-topping hits.", 'https://yt3.googleusercontent.com/7tCfeCWH4arhsTM-4Rz4IxWieQbegzibeXlG-kbytAujdk5dr2K0gBb8NG0Cvk6lB1dPkjyd=s900-c-k-c0x00ffffff-no-rj'),
+  ],
+};
+
+// Dynamically add featured artists
+const artistCard = (artistData, index) => {
+  const {
+    fullName, songs, blurb, imgUrl,
+  } = artistData;
+  const songsString = songs.reduce((acc, song, i) => {
+    if (i === songs.length - 1) {
+      return `${acc + song} `;
+    }
+    return `${acc + song}, `;
+  }, '');
+
+  return `
+    <div class="card card--${index}">
+      <figure class="portrait">
+        <svg viewBox="0 0 100 100" class="checkers">
+          <use xlink:href="sprite.svg#checkers"></use>
+        </svg>
+        <img src="${imgUrl}" alt="${fullName}">
+      </figure>
+      <div class="profile">
+        <h3>${fullName}</h3>
+        <h5>Hits: ${songsString}</h5>
+        <p class="description">${blurb}</p>
+      </div>
+    </div>
+  `;
+};
+const featuredCards = grab('.cards');
+const { artists } = db;
+const featuredArtists = artists.reduce((acc, artist, i) => acc + artistCard(artist, i), '');
+render(featuredCards, featuredArtists);
+
+// Show only 2 artists in mobile
+const artistCards = Object.values(featuredCards.children);
+const visibleCards = [0, 1];
+const renderMobileCards = () => {
+  artistCards.forEach((card, i) => {
+    if (visibleCards.includes(i)) {
+      card.classList.remove('hide');
+    } else {
+      card.classList.add('hide');
+    }
+  });
+
+  if (visibleCards[0] === 4) {
+    visibleCards[0] = 0;
+    visibleCards[1] = 1;
+  } else {
+    visibleCards[0] += 2;
+    visibleCards[1] += 2;
+  }
+};
+
+renderMobileCards();
+
+// activate button to cicle through featured artists on mobile version
+const moreArtists = grab('.more');
+onClick(moreArtists, renderMobileCards);
+
+// Toggle mobile menu
+const menuBurger = grab('.menu-burger');
+const logo = grab('.logoBox');
+const mainNavItems = Object.values(grab('.main').children);
+const getTicket = grab('.getTicket');
+const header = grab('header');
+const toggleMenu = () => {
+  const { className } = header;
+  if (className === 'header') {
+    header.className = 'openNav';
+  } else {
+    header.className = 'header';
+  }
+};
+
+onClickAll([...mainNavItems, menuBurger, logo, getTicket], toggleMenu);
```

Comment:
- [ ] Well done for implementing the suggestion of the previous reviewer 👏 However, I noticed that on desktop version, while navigation to about from home, the mobile menu is being displayed and vice versa 🐛 Since, this issue was not present in the last review, kindly rectify this. Also, when on clicking any link (home or about), the mobile menu is appearing. Please have a look at the video below for better clarity 👇




https://github.com/Drod86/reggaeducaton/assets/106119105/b2f72592-3003-491f-b7c6-1ee6ab74b6c0

Reply:
Thank you @KanzaTahreem for catching this oversight. I have corrected this error.